dex/elrond_dex_pair/src/lib.rs

```diff
@@ -317,6 +317,56 @@ pub trait Pair:
             .into())
     }
 
+    #[payable("*")]
+    #[endpoint(removeLiquidityAndBuyBackAndBurnToken)]
+    fn remove_liquidity_and_burn_token(
+        &self,
+        #[payment_token] token_in: TokenIdentifier,
+        #[payment_amount] amount_in: Self::BigUint,
+        token_to_buyback_and_burn: TokenIdentifier,
+    ) -> SCResult<()> {
+        require!(
+            !self.lp_token_identifier().is_empty(),
+            "LP token not issued"
+        );
+        require!(
+            token_in == self.lp_token_identifier().get(),
+            "Wrong liquidity token"
+        );
+
+        let first_token_id = self.first_token_id().get();
+        let second_token_id = self.second_token_id().get();
+
+        let first_token_min_amount = 1u64.into();
+        let second_token_min_amount = 1u64.into();
```

Comment:
You can use the get equivalent for liquidity view function from pair to determine these values.

Reply:
What benefit would that bring?